client/src/components/ButtonBacktoTop/ButtonBacktoTop.css

```diff
@@ -0,0 +1,5 @@
+.TopButton {
```

Comment:
You should do it with tailwind instead of making a new css file.

Reply:
Okay,